kitchen.recipes.yml

```diff
@@ -9,7 +9,8 @@ verifier:
   name: inspec
   inspec_tests:
     # recipe tests will use controls from these directories
-    - test/recipes
```

Comment:
Please keep this line and remove the 2 lines below. Here we define the folder we peek controls from, below we define controls to pick.

Reply:
yes it was my idea. I discussed with @francesco-giordano to find the best solution and I think we found one. I'm going to update this patch.